tests/topotests/bgp_srv6_sid_explicit/r1/frr.conf

```diff
@@ -7,6 +7,7 @@ log commands
 password zebra
 !
 ipv6 route 2001:db8:2:2::/64 2001:db8:12::2
+ipv6 route 2001:db8:3:2::/64 2001:db8:13::2
```

Comment:
After fixing the locator prefix and the loopback address of r3, make sure to update this route accordingly.

Reply:
ok